### PR TITLE
Use Debian base to build Cluster Autoscaler image

### PR DESCRIPTION
**Pull request by aleksandra-malinowska**
Use Debian base to build Cluster Autoscaler image.

### CODE REVIEW

File: builder/Dockerfile [hunk 1]
```diff
@@ -12,11 +12,10 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM gcr.io/google-containers/ubuntu-slim:0.8
-MAINTAINER Marcin Wielgus "mwielgus@google.com"
+FROM gcr.io/google-containers/debian-base-amd64:0.1
```
Comment (tallclair) [inline builder/Dockerfile]: Please use 0.3 (the latest)

Comment (aleksandra-malinowska) [reply builder/Dockerfile]: Done

File: cluster-autoscaler/Dockerfile [hunk 2]
```diff
@@ -12,11 +12,11 @@
...
-FROM gcr.io/google-containers/ubuntu-slim:0.8
-MAINTAINER Marcin Wielgus "mwielgus@google.com"
+FROM gcr.io/google-containers/debian-base-amd64:0.3
+LABEL maintainer="Marcin Wielgus <mwielgus@google.com>"
 
-RUN apt-get update && apt-get install --yes ca-certificates \
-  && apt-get clean \
+ENV DEBIAN_FRONTEND noninteractive
+RUN clean-install ca-certificates \
   && rm -rf /var/lib/apt/lists/*
```
Comment (tallclair) [inline cluster-autoscaler/Dockerfile]: clean-install handles this, no need to duplicate here.

File: builder/Dockerfile [hunk 3]
```diff
@@ -12,11 +12,10 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM gcr.io/google-containers/ubuntu-slim:0.8
-MAINTAINER Marcin Wielgus "mwielgus@google.com"
+FROM gcr.io/google-containers/debian-base-amd64:0.3
```
Comment (tallclair) [inline builder/Dockerfile]: The slim-ness of the build image probably doesn't matter as much - have you considered just using the official go images? (e.g. golang:1.8.3). This is how kubernetes releases [are built](https://github.com/kubernetes/kubernetes/blob/master/build/build-image/cross/Dockerfile).

Comment (aleksandra-malinowska) [reply builder/Dockerfile]: Done

File: cluster-autoscaler/Dockerfile (hunk 2 above)
Comment (aleksandra-malinowska) [reply cluster-autoscaler/Dockerfile]: removed

**Pull request merged**